enstore-storage/enstore-storage

```diff
@@ -253,6 +253,8 @@ date cannot be null.
         #end_time = timeutil.at_minute(datetime.datetime.now())
         end_time = None
         # Give to the end time a 60 seconds delay. To allow for enstore DB processing time
+        if limit is None:
+            limit = 0
         if limit > 0:
```

Comment:
But i think we can still clean it up a bit with

```suggestion
        if limit is not None and limit > 0:
```

or just

```suggestion
        if limit and limit > 0:
```

...

Or if we happen to know that limit is not negative (I do not know whether that is a safe assumption), just

```suggestion
        if limit:
```